config.md

```diff
@@ -241,7 +241,9 @@
 
 |Key|Description|Type|Default Value|
 |---|-----------|----|-------------|
-|errorHistoryCount|The number of historical errors to retain in the operation|`int`|`25`
+|maxHistoryActions|The number of actions to store per historical status updates|`int`|`50`
+|maxHistoryCount|The number of historical status updates to retain in the operation|`int`|`50`
+|maxHistorySummaryCount|The number of historical status summary records to retain in the operation|`int`|`50`
```

Comment:
I wonder if we really need this configuration parameter, as this should be bounded by the number of types of status we have.